src/styles/styles.css

```diff
@@ -14,7 +8,7 @@ html {
 }
 
 body {
-	font-family: 'Montserrat', Montserrat, sans-serif;
```

Comment:
I'll pick us a pretty Google font. I'm also thinking of removing the star background and just going with a color. Stay tuned for redesign

Reply:
That'd be great - looking forward to a new UI